src/package-request.js

```diff
@@ -404,7 +403,6 @@ export default class PackageRequest {
     const isDepOld = ({current, latest, wanted}) =>
       latest === 'exotic' || (latest !== 'exotic' && (semver.lt(current, wanted) || semver.lt(current, latest)));
```

Comment:
I thought this deserves a quick cleanup `latest === 'exotic' || (semver.lt(current, wanted) || semver.lt(current, latest));`

Reply:
Oh geez I'm sorry... I should read the thread first huh :)
I'll review yours and kill mine???

Reply:
oh i don't mind it at all. Yours has a test so let's go with that! I can close mine :)

Reply:
Actually, since mine has a unit test, should we keep mine? I'll pull in your change above too... Sorry for the churn... just physically got back from vacation, but my brain is still at the beach 🌴 ...